locales/ruRU.lua

```diff
@@ -675,18 +675,29 @@ SPELL_STAT5_NAME = "Spirit"
 --
 
 L["statList"] = {
+	{pattern = "сила атаки", id = ATTACK_POWER},
 	{pattern = "силу атаки", id = ATTACK_POWER},
+	{pattern = "силы атаки", id = ATTACK_POWER},
+	{pattern = "силы вашей атаки", id = ATTACK_POWER},
 	{pattern = "к силе атаки", id = ATTACK_POWER},
+	{pattern = "силам природы", id = NATURE_RES},
+	{pattern = "сила заклинаний", id = SPELL_DMG},
+	{pattern = "сила ваших заклинаний", id = SPELL_DMG},
+	{pattern = "силу заклинаний", id = SPELL_DMG},
+	{pattern = "силы заклинаний", id = SPELL_DMG},
+	{pattern = "к силе заклинаний", id = SPELL_DMG},

```

Comment:
These translations are only for "Stat Breakdowns" where RatingBuster adds additional information in parentheses after the stat. Resistance and Spell Damage aren't currently broken down into anything in any locale.

If you meant for these to be used for summaries, we'd have to add them to StatLogic, like you did in #127 .

Reply:
You did not understand. They are needed not for summaries, but for proper working of priorities. If you will not add these, nature resistance will be converted as strength because it contains the word "сила" in "силам природы". Same with spell damage - it also contains the word "сила" and all spell damage will be converted as strength. 
![NR](https://user-images.githubusercontent.com/61734903/205896427-04a5585c-c1da-4189-9ce8-c7b712bd8060.jpg)
![spd](https://user-images.githubusercontent.com/61734903/205896482-67d3189c-b34a-40b7-9e0b-ce02d79781b5.jpg)

Reply:
I see, thanks for elaborating. I'll just add a comment explaining them then.